code/client/src/pages/Create.jsx

```diff
@@ -273,7 +281,7 @@ const Create = () => {
           <Heading>You are all set!</Heading>
           <Space direction='vertical' size='small'>
             <Hint>Wallet Address</Hint>
-            <Text>{address}</Text>
+            <Text>{address && getAddress(address).bech32}</Text>
```

Comment:
The implementation of `getAddress` seems to be quite slow. The first thing it does seems to be a regex matching https://github.com/harmony-one/sdk/blob/3c7061887f25c6ba8def5af05672cdc4f44eb2dc/packages/harmony-utils/src/validators.ts#L6 , followed by multiple passes of the 20 bytes array and many string operations and comparisons.  It may be too slow to be in the rendering loop. We might need another state variable for it, watched by useEffect and only converted when it is at full length.